### PR TITLE
fvntr: adding more speakers

### PR DESCRIPTION
**Pull request by fvntr**
Adding confirmed speakers

---

> ✅️ By submitting this PR, I have verified the following

- [x] Checked to [see if a similar PR has already been opened](https://github.com/techqueria/website/pulls) 🤔️
- [x] Reviewed the [contributing guidelines](https://github.com/techqueria/website/blob/master/.github/CONTRIBUTING.md) 🔍️
- [x] Added my name to the bottom of the list under the **Contributors** section in the [README.md](https://github.com/techqueria/website/blob/master/README.md) with a link to my personal website or GitHub profile 👥️ (Optional)

### CODE REVIEW

File: data/speakers.json [hunk 1]
```diff
@@ -100,5 +100,68 @@
...
+    "title": "Career Services Manager",
+    "organization": "Flockjay",
+    "image": "/assets/img/team/laura-dominguez.jpg",
+    "linkedin": "https://www.linkedin.com/in/laura-doming/"
+  },
+  "Renee Ramirez": {
+    "name": "Renee Ramirez",
+    "title": "Executive Advisor",
+    "organization": "Google",
+    "image": "",
```
Comment (fvcproductions) [inline data/speakers.json:115]: image should be here

**Comment by fvcproductions** [comment]
the images need to be square size so 200x200 or 400x400

Screen Shot 2020-09-10 at 7 30 32 PM

**Comment by fvcproductions** [comment]
@fvntr can u update the image so these changes get reflected

**Comment by fvntr** [comment]
resized images to 400x400

**Comment by fvntr** [comment]
ok verified, all pics look good

**Pull request merged**